src/server/csp.test.ts

```diff
@@ -0,0 +1,55 @@
+import { ElementType } from '@guardian/content-api-models/v1/elementType';
+import { getThirdPartyEmbeds } from 'capi'
+import { articleContentWith } from 'item.test';
```

Comment:
By `import`ing the `item.test.ts` file here, will all the tests in that file also run (because they're side-effects)? If so, won't that mean that we end up running those tests twice? Once when Jest finds and runs `item.test.ts`, and once when it runs `csp.test.ts`.

Reply:
@JamieB-gu Im not entirely sure whether or how this will run twice? i can suggest to directly place the function from item.test.ts into the cspt.test file to ensure no cross over/side effects?

Reply:
Just looked through again, I would have thought that it would merely call the standalone function once for the item tests and once for the csp.
The function would merely return a `body`. unless I'm over simplifying here?

Reply:
We've discussed this offline, but to briefly summarise here for posterity:

1. Jest discovers and runs `csp.test.ts`. `csp.test.ts` `import`s `item.test.ts`. `item.test.ts` contains a number of tests. When `csp.test.ts` imports `item.test.ts`, those tests run as side effects of the `import`. That's the first time they run.

2. Jest then discovers and runs `item.test.ts`. `item.test.ts` contains a number of tests that are run. That's the second time they run.